test/worker_parallel_scheduling_test.py

```diff
@@ -149,7 +149,7 @@ def test_raise_unpicklable_exception_in_complete(self, send):
             UnpicklableExceptionTask().complete()
         except Exception as e:
             ex = e
-        self.assertRaises(pickle.PicklingError, pickle.dumps, ex)
+        self.assertRaises(Exception, pickle.dumps, ex)
```

Comment:
It would have been better to do `self.assertRaises((pickle.PicklingError, AttributeError), pickle.dumps, ex)` to ensure that we weren't catching too broadly.